test/Operators/remapping.jl

```diff
@@ -64,6 +68,10 @@ function consistent(R::LinearRemap)
     return row_sums ≈ v
 end
 
+function locally_consistent(R::LinearRemap)
+
+end
```

Comment:
Will probably split `locally_conservative` and `locally_consistent` into a separate PR that adds other test enhancements. I want to add some tests of non-square domains for existing 2D cases, some tests that remapping from a space to itself builds the identity, and perhaps tests that non-overlapping meshes return a matrix of zeros.

Additions to the test suite for this PR will focus solely on some basic 2D SE <-> SE cases checking conservation, consistency, and sparsity pattern (which I think is a good stand in for checking against `R_true` in non-trivial cases).